CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Node.js Buildpack Changelog
 
 ## master
+- update git important bits url ([#794](https://github.com/heroku/heroku-buildpack-nodejs/pull/794))
```

Comment:
Can you specify exactly what this is doing? Something like "Update docs URL when node modules are checked into git"?